src/DsaColorPrinter.cc

```diff
@@ -0,0 +1,647 @@
+#include "llvm/ADT/GraphTraits.h"
```

Comment:
Why do we need another class? 
We should try to extend `DsaPrinter` that takes optionally a coloring map and uses it if available.